addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/thing-types.xml

```diff
@@ -85,6 +87,24 @@
 		</channels>
 	</channel-group-type>
 
+	<channel-group-type id="netMenuControls">
+		<label>Net/USB Menu</label>
+		<channels>
+            <channel id="control" typeId="netControl"/>
+			<channel id="item0" typeId="menu0"/>
```

Comment:
Might I ask why you have defined 10 different channel types? They all seem to be instances of the same type.
Note that you can override the label&description of the type here in the channel definition, if this was your reason.